src/guide/mutators.md

```diff
@@ -33,6 +33,7 @@ The Unwrap* mutator family will unwrap function parameters.
 | UnwrapArrayReverse | `$a = array_reverse(['A', 'B', 'C']);` | `$a = ['A', 'B', 'C'];` |
 | UnwrapStrRepeat | `$a = str_repeat('A', 3);` | `$a = 'A';` |
 | UnwrapStrToLower | `$a = strtolower('Hello!');` | `$a = 'Hello!';` |
+| UnwrapStrToUpper | `$a = strtoupper(['A', 'B', 'C']);` | `$a = ['A', 'B', 'C'];` |
```

Comment:
could you please use string as a parameter?

Reply:
@borNfreee  I updated the documentation